### DISCUSSION TITLE
Can someone please re-activate my Slack account?

### DISCUSSION THREAD

**Discussion by djalbat** [babel/babel#12646] [General] [answered]
Hi!

I am sorry about this. Can someone please re-activate my `james.smith@djalbat.com` account for your Slack workspace?

Many thanks and kind regards,
James

**Comment by nicolo-ribaudo** [comment, marked as answer]
Done, not sure why it was deactivated.

**Comment by djalbat** [comment]
Many thanks, Nicolo. Chances are I deactivated it myself.
